features/step_definitions/common_steps.rb

```diff
@@ -181,10 +181,9 @@
   breadcrumb.text().should == breadcrumb_text
 end
 
-Then /^I (don't |)see the '(.*)' link$/ do |negative, link_text|
-  page.should have_link(link_text) if negative.empty?
-
-  page.should_not have_link(link_text) unless negative.empty?
+Then /^I (don't |)see the '(.*)' (button|link)$/ do |negative, selector_text, selector_type|
+page.should have_selector(:link_or_button, selector_text) if negative.empty?
```

Comment:
It will only accept either a link or a button, no other selector, here's an overview of this logic with references:
https://stackoverflow.com/questions/13865625/capybara-matcher-for-presence-of-button-or-link